src/main/kotlin/io/perfometer/statistics/consumer/statisticsConsumer.kt

```diff
@@ -0,0 +1,16 @@
+package io.perfometer.statistics.consumer
+
+import io.perfometer.statistics.ScenarioSummary
+
+internal fun getStatisticsConsumer(output: Output): (ScenarioSummary) -> Unit {
+    return when (output) {
+        Output.STDOUT -> { summary: ScenarioSummary -> StdOutStatisticsPrinter().print(summary) }
+        Output.TEXT_FILE -> { summary: ScenarioSummary -> devNull(summary) }
+        Output.PDF -> { summary: ScenarioSummary -> devNull(summary) }
+        Output.HTML -> { summary: ScenarioSummary -> devNull(summary) }
+    }
+}
+
+private fun devNull(scenarioSummary: ScenarioSummary) {
+    // not implemented
```

Comment:
It is better to use TODO function to throw exception, not silently ignoring error.

Reply:
:+1: